docs/source/llms/deployments/index.rst

```diff
@@ -1187,6 +1187,46 @@ MLflow Deployments Server API Documentation
 
 `API documentation <./api.html>`_
 
+OpenAI Compatibility
+====================
+
+MLflow Deployments Server is compatible with OpenAI's API and supports the ``chat``, ``completions``, and ``embeddings`` APIs.
```

Comment:
```suggestion
MLflow Deployments Server is compatible with OpenAI API and supports the ``chat``, ``completions``, and ``embeddings`` APIs.
```

super-nit: OpenAI's API seems to be not very common wording

Reply:
👍